client/config/webpack.config.js

```diff
@@ -21,6 +22,10 @@ import ForkTsCheckerWebpackPlugin from "fork-ts-checker-webpack-plugin";
 // Source maps are resource heavy and can cause out of memory issue for large source files.
 const shouldUseSourceMap = process.env.GENERATE_SOURCEMAP !== "false";
 
+// to compare file sizes in PRs we need them to not include hashes
```

Comment:
This comment seems out of context?

Reply:
Not sure what you mean: it explains the need for the `analyzeBundlePR` option

Reply:
Gotcha!